src/hotspot/share/opto/node.cpp

```diff
@@ -2208,25 +2202,22 @@ bool PrintBFS::parse_options() {
   return true;
 }
 
-bool PrintBFS::filter_category(Node* n, Filter& filter) {
+bool PrintBFS::filter_category(const Node* n, Filter& filter) {
```

Comment:
Maybe you could add a method comment that you are not filtering on the category for `Mixed` but actually look at the outputs of it and also consider `is_CFG()`.